### ISSUE TITLE
bind-9.10 uses /etc/ instead of /opt/local/etc

### ISSUE THREAD

**Issue by bahamat** [TritonDataCenter/pkgsrc-legacy#238]
Bind 9.10 defines `/etc` as `sysconfigdir` while bind-9.9 uses `/opt/local/etc`. This means that `named.conf` and `rndc.key` must bin in `/etc`.

Bind 9.9 also shipped a stub `/opt/local/etc/named.conf`. Bind 9.10 doesn't include one, but the SMF has a dependency on `/opt/local/etc/named.conf` (and no dependency on `/etc/named.conf`) but no `named.conf` file.

**Comment by bahamat** [comment]
The following patch fixes the issue:

``` diff
diff --git a/net/bind910/Makefile b/net/bind910/Makefile
index 0ec082d..9c28f82 100644
--- a/net/bind910/Makefile
+++ b/net/bind910/Makefile
@@ -28,7 +28,7 @@ GNU_CONFIGURE=                yes
 #CONFIG_SHELL=         sh -x

 CONFIGURE_ARGS+=       --with-libtool
-CONFIGURE_ARGS+=       --sysconfdir=/etc
+CONFIGURE_ARGS+=       --sysconfdir=${PKG_SYSCONFDIR}
 CONFIGURE_ARGS+=       --localstatedir=${VARBASE}
 CONFIGURE_ARGS+=       --disable-openssl-version-check
 CONFIGURE_ARGS+=       --with-openssl=${SSLBASE:Q}
```

It would greatly help me if this could be merged into `2014Q3` and later branches.

**Comment by bahamat** [comment]
This fix was released into release/2014Q3 with bind-9.10.0pl2nb1.

**Comment by mamash** [comment]
Fixed as of 2014Q3.
